### ISSUE TITLE
change anchor size and stride

### ISSUE THREAD

**Issue by huxianxian** [fizyr/keras-retinanet#497]
How to change the anchor size and stride for my own dataset? When I try to do this , i always get a incompitable shape [10] vs [5].

**Comment by hgaiser** [comment]
Why do you want to change the anchor sizes and stride? They're dependent on how the network downsamples the image and at least for the backbones currently in keras-retinanet they are correct (to the best of my knowledge). I don't think they should change, unless you're also changing the backbone.

**Comment by hgaiser** [comment]
Closing due to inactivity.

**Comment by hgaiser** [comment]
Sure, it'd do the same thing.

**Comment by JinwenJay** [comment]
@hgaiser. Hi, I have changed the anchor ratios, scales and pyramid_levels so that there is no red box after running the debug.py, and actually, I made it. But after the changes, training does't work anymore, because the loss never decrease during training. Did I should not change these? Or where is another place I should also change?
